src/Maker/MakeTest.php

```diff
@@ -106,14 +108,40 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
             if (!isset(self::DESCRIPTIONS[$type])) {
                 throw new RuntimeCommandException(sprintf('The test type must be one of "%s", "%s" given.', implode('", "', array_keys(self::DESCRIPTIONS)), $type));
             }
+        } else {
+            $input->setArgument(
+                'type',
+                $io->choice('Which test type would you like?', self::DESCRIPTIONS)
+            );
+        }
 
-            return;
+        if ('ApiTestCase' === $input->getArgument('type') && !class_exists(ApiTestCase::class)) {
+            $io->warning([
+                'API Platform is required for this test type. Install it with',
+                'composer require api',
+            ]);
```

Comment:
Unfortunately, the dependency system is called before the interactive step. So previously, no error or warning was shown. That's actually ok - I like showing a warning better than exploding with an error that they must stop and run the `composer require` command before starting the `make:test` command again.

Reply:
Agreed, seeing as we're not actually collecting a bunch of input that is dependent on `Api Platform` at this time - the warning makes sense..